lib/commandCenter/commandCenter.js

```diff
@@ -388,15 +387,52 @@ function compress(content, cb) {
 	});
 }
 
+/**
+ * Promisify the user command execute method, or
+ * ensure its return value is passed to state.respond
+ *
+ * Once callback-based user commands are removed from MAGE,
+ * you will be able to remove this method.
+ *
+ * @param {*} cmdInfo
+ * @param {*} paramList
```

Comment:
`state` is not mentioned in the params

Reply:
Not sure what types would be valid, but will add State